lmms_eval/models/llava.py

```diff
@@ -58,6 +58,7 @@ def __init__(
         device_map="cuda:0",
         conv_template="vicuna_v1",
         use_cache=True,
+        tie_weights: bool = True,
```

Comment:
Should this be true for llava? At least for v1.5 (and llama 2 models in general), this shouldn't be true: https://huggingface.co/llava-hf/llava-1.5-13b-hf/commit/d23149f1544974682263229a9b373e6e0395826d

Reply:
hm, I think we should merge in and then just make sure to set tie_weights to false if we want to?